backend/app/expenses/service.py

```
@@ -0,0 +1,1095 @@
+            "expense": expense,
+            "relatedSettlements": settlements
+        }
+
+    async def update_expense(self, group_id: str, expense_id: str, updates: ExpenseUpdateRequest, user_id: str) -> ExpenseResponse:
+        """Update an expense"""
+        
+        try:
+            # Validate ObjectId format
+            try:
+                expense_obj_id = ObjectId(expense_id)
+            except Exception as e:
```

Flake8 (7.2.0)

[error] local variable 'e' is assigned to but never used

(F841)